### PR TITLE
[feature] Add the compilation and execution times of an XQuery to the wrapped output from the REST Server

### PR DESCRIPTION
**Pull request by adamretter**

### CODE REVIEW

File: src/org/exist/storage/serializers/Serializer.java [hunk 1]
```diff
@@ -894,14 +896,17 @@ public void toSAX(NodeProxy p) throws SAXException {
...
-         * @param typed Indicates whether the output types should be wrapped
-	 * @throws SAXException
+	 * @param typed Indicates whether the output types should be wrapped
+	 * @param compilationTime The time taken to compile the query which produced the sequence
+	 * @param executionTime The time taken to execute the query which produced the sequence
+	 *
+	 * @throws SAXException If an error occurs during serialization
 	 */
-	public void toSAX(Sequence seq, int start, int count, boolean wrap, boolean typed) throws SAXException {
+	public void toSAX(final Sequence seq, int start, final int count, final boolean wrap, final boolean typed, final long compilationTime, final long executionTime) throws SAXException {
```
Comment (dizzzz) [inline src/org/exist/storage/serializers/Serializer.java]: extending an existing method is often a bad thing..... But there are clearly no other users?

**Comment by dizzzz** [comment]
looks good to me....  no test?

**Comment by dizzzz** [comment]
```
BUILD FAILED
/home/travis/build/eXist-db/exist/build/scripts/junit.xml:125: Test org.exist.xquery.functions.request.GetHeaderTest failed
```

**Comment by adamretter** [comment]
@dizzzz Sorry I missed a commit, so I just rebased and pushed.

Testing timings is hard as they are non-deterministic.

**Pull request merged**